docs/UserGuide.md

```diff
@@ -11,6 +11,25 @@ AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized fo
 <!-- * Table of Contents -->
 <page-nav-print />
 
+--------------------------------------------------------------------------------------------------------------------
+## Glossary
+
+- [Quick start](#quick-start)
+- [Features](#features)
+  - [Viewing help : `help`](#viewing-help--help)
+  - [Adding a person : `add`](#adding-a-person-add)
+  - [Listing all persons : `list`](#listing-all-persons--list)
+  - [Editing a person : `edit`](#editing-a-person--edit)
+  - [Locating a persons by name, phone_number, room_number or tag](#locating-persons-by-name-phone_number-room_number-or-tag-find)
```

Comment:
ah, why don't we just use the same format as the other lines like.....[Locating a persons by name, phone_number, room_number or tag: `find`]